src/helm/proxy/clients/megatron_client.py

```diff
@@ -0,0 +1,95 @@
+import json
+import requests
+from typing import Any, Dict, List
+import traceback
+
+from helm.common.cache import CacheConfig
+from helm.common.request import EMBEDDING_UNAVAILABLE_REQUEST_RESULT, Request, RequestResult, Sequence, Token
+from helm.common.tokenization_request import TokenizationRequest
+from helm.proxy.clients.huggingface_client import HuggingFaceClient
+from helm.proxy.clients.client import Client, wrap_request_time, truncate_sequence
+
+
+class MegatronClient(HuggingFaceClient):
```

Comment:
Could you add a docstring explaining how this is supposed to be used i.e.
Link to the repo https://github.com/NVIDIA/Megatron-LM and say that HELM expects `tools/run_text_generation_server.py` to be run on `localhost:5000`

Reply:
Done.